packages/fetchai/connections/p2p_libp2p/aea/envelope.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package aea;
```

Comment:
We need to come up with a proper package naming convention

Reply:
Also relates to protocols we generate for AEA to AEA comms

Reply:
Agree